configure.ac

```diff
@@ -1448,16 +1448,18 @@ fi
 # Configuring pmi, pmilib, and pm
 #
 
-AC_ARG_WITH(pmi,
-	AS_HELP_STRING([--with-pmi=name], [Specify the pmi interface for MPICH]),,
-	with_pmi=pmi1)
-
-AC_ARG_WITH(pmilib,
-	AS_HELP_STRING([--with-pmilib=name],
-                [Specify the PMI library. Deprecated. Use --with-pmi, or
-                 --with-pmi2, or --with-pmix to specify library path.
-                ]),,
-	with_pmilib=mpich)
+AC_ARG_WITH(pmi, [
+  --with-pmi=name - Specify the pmi interface for MPICH.
+                      pmi1 - use PMIv1 (default)
+                      pmi2 - use PMI2
+                      pmix - use PMIx
+],, with_pmi=pmi1)
+
+AC_ARG_WITH(pmilib, [
+  --with-pmilib=option -  [Deprecated] Specify the PMI library. Use --with-pmi=[PATH],
```

Comment:
What is the preferred way to specify the path to an external PMI library? When setting up Polaris CI, I still ended up using this option to point to link with the Cray PMI library.

Reply:
Does `--with-pmi2=[cray-lib-path]` work?

Reply:
> Does `--with-pmi2=[cray-inc-lib-path]` work?

It does after I rebased onto the latest `main`. I got bitten using an older branch.